app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
users.scss and users.coffee can be safely removed to keep app clutter-free of unused files!